src/cpuid/src/transformer/common.rs

```diff
@@ -36,7 +36,10 @@ pub fn update_feature_info_entry(
     let max_cpus_per_package = u32::from(common::get_max_cpus_per_package(vm_spec.cpu_count)?);
 
     // X86 hypervisor feature
-    entry.ecx.write_bit(ecx::HYPERVISOR_BITINDEX, true);
+    entry
+        .ecx
+        .write_bit(ecx::TSC_DEADLINE_TIMER_BITINDEX, true)
+        .write_bit(ecx::HYPERVISOR_BITINDEX, true);
```

Comment:
AMD doesn't support this feature. If we enable it in the guest, the boot time we'll decrease, but we'll sacrifice runtime performance. I think it's better to have different expected values for Intel and AMD for the boot performance test.

Reply:
Synced offline with Serban. Indeed, different expected values sounds like a good workaround to me.

Reply:
I gathered some more info on the TSC_DEADLINE_TIMER. The deadline TSC is fully emulated by KVM even on Intel where this timer is hardware supported. You can check the emulation code for this [here](https://github.com/torvalds/linux/blob/8f02f363f76f99f08117336cfac7f24c76b25be3/arch/x86/kvm/x86.c#L3201). So, whether the hardware supports it or not, is pretty irrelevant given that KVM implements it nevertheless. 
There was actually another bug expressing the same thing here: https://bugs.launchpad.net/ubuntu/+source/linux/+bug/1724912.
Another thing relevant here is the dmesg output. You can see the following message:
```
[    0.000000] Hypervisor detected: KVM
[    0.000000] tsc: Fast TSC calibration using PIT
```
So after the system detects that is runs under a hypervisor, calibration happens nevertheless. So, this deems this code [redundant](https://github.com/torvalds/linux/blob/9e98c678c2d6ae3a17cb2de55d17f69dddaa231b/arch/x86/kernel/apic/apic.c#L821).
@serban300 please let me know if any info is needed.
Thanks!

Reply:
You're right. Looks like this doesn't lead to any performance penalty. LGTM.